kani-compiler/src/codegen_cprover_gotoc/codegen/place.rs

```diff
@@ -242,54 +240,61 @@ impl<'tcx> GotocCtx<'tcx> {
     fn codegen_field(
         &mut self,
         parent_expr: Expr,
-        parent_ty_or_var: TypeOrVariant<'tcx>,
-        field: &FieldIdx,
-        field_ty_or_var: TypeOrVariant<'tcx>,
+        parent_ty_or_var: TypeOrVariant,
+        field_idx: FieldIdx,
+        field_ty_or_var: TypeOrVariant,
     ) -> Result<Expr, UnimplementedData> {
         match parent_ty_or_var {
             TypeOrVariant::Type(parent_ty) => {
                 match parent_ty.kind() {
-                    ty::Alias(..)
-                    | ty::Bool
-                    | ty::Char
-                    | ty::Int(_)
-                    | ty::Uint(_)
-                    | ty::Float(_)
-                    | ty::FnPtr(_)
-                    | ty::Never
-                    | ty::FnDef(..)
-                    | ty::CoroutineWitness(..)
-                    | ty::Foreign(..)
-                    | ty::Dynamic(..)
-                    | ty::Bound(..)
-                    | ty::Placeholder(..)
```

Comment:
What happened to `ty::Placeholder`?

Reply:
Great question, it doesn't exist today. This type cannot appear in the MIR level that StableMIR represents, so we just didn't add it.

Reply:
Thanks for the explanation.